src/main/resources/open-api-documentation/openapi.json

```diff
@@ -0,0 +1,306 @@
+{
```

Comment:
The resource file for swagger may to be maintained, it is possible to use swagger annotations to prepare API documentation?

Reply:
At the beginning I was using Annotations but founded it more complicate than using just Swagger editor and export it into JSON file 
Then we can use this JSON in Swagger editor or Postman